modules/compliance-scheduling-pods-with-resource-requests.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/compliance-scans.adoc
+
+:_content-type: CONCEPT
+[id="compliance-scheduling-pods-with-resource-requests"]
+= Scheduling Pods with resource requests
+
+When a Pod is created, the scheduler selects a Node for the Pod to run on. Each node has a maximum capacity for each resource type in the amount of CPU and memory it can provide for the Pods. The scheduler ensures that the sum of the resource requests of the scheduled containers is less than the capacity nodes for each resource type.
```

Comment:
I don't think `Node` needs to be capitalized here

Reply:
+1